README.md

```diff
@@ -0,0 +1,136 @@
+Welcome to Corkscrew
+====================
+
+## Introduction
+
+Corkscrew is a tool for tunneling SSH through HTTP proxies, but... you might 
+find another use for it.
+
+Corkscrew has been compiled on:
+
+- HPUX 
+- Solaris
+- FreeBSD
+- OpenBSD
+- Linux
+- Win32 (with Cygwin)
+
+Corkscrew has been tested with the following HTTP proxies:
+
+- Gauntlet
+- CacheFlow
+- JunkBuster
+- Apache mod_proxy
+
+Please open a pull request if you get it working on other proxies or compile
+it elsewhere.
+
+
+## Where Do I Get It?
+
+Corkscrew's primary distribution site was 
+[agroman.net/corkscrew](https://web.archive.org/web/20170510154150/http://agroman.net/corkscrew/), 
+however it seems that the site went down and this repository is here to keep
+the code available. The new location is then 
+[github.com/bryanpkc/corkscrew](https://github.com/bryanpkc/corkscrew).
+
+
+## How Do I Install It?
+
+First you need to install development tools:
```

Comment:
Official source releases would have the autoconf stuff pre-generated, therefore typically all people need to do is following the INSTALL instructions. I suggest replacing this line with the following:
```markdown
If the `configure` shell script already exists in the source distribution, simply run:

    ./configure
    make install

Refer to the INSTALL file for more information.

If the `configure` shell script does not exist, you have probably just cloned the source code from GitHub.
You will need the following dependencies to build the project with GNU autoconf.
```

Reply:
Well, there is no official source release right?

Maybe we can keep it this way and then put it in a different section when official releases are being reinstated?

Reply:
Fair enough.